src/components/Colorbox.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+
+const colors = {
+  items: 28,
+  cssColors: [
+    "Black",
+    "DarkGrey",
+    "DarkRed",
+    "Olive",
+    "Green",
+    "SteelBlue",
+    "DarkBlue",
+    "Purple",
+    "DarkKhaki",
+    "LightGreen",
+    "RoyalBlue",
+    "Indigo",
+    "Navy",
+    "Saddlebrown",
+    "White",
+    "LightGrey",
+    "Red",
+    "Yellow",
+    "Lime",
+    "Skyblue",
+    "Blue",
+    "Violet",
+    "Khaki",
+    "LawnGreen",
+    "Skyblue",
+    "DarkViolet",
+    "Fuchsia",
+    "Chocolate"
+  ]
+};
+
+const Colorbox = () => {
```

Comment:
I would call it ColorSelector instead of ColorBox because it is not a box with a colour.

Reply:
I agree with you. That is a better name for this component, I didn't think it that way.